package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "soap",
-  "version": "0.43.0",
```

Comment:
please revert the version field.

Reply:
Ok.
Although I believe it is correct according to semver specification [https://semver.org/#spec-item-7](https://semver.org/#spec-item-7)
```Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API```